spec/pg/decoders/range_decoder_spec.cr

```diff
@@ -0,0 +1,10 @@
+require "../../spec_helper"
+
+describe PG::Decoders do
+  describe "int4range" do
+    test_decode "exclusive", "'[10,20)'::int4range", Range.new(10, 20)
+    test_decode "exclusive", "'(10,20]'::int4range", Range.new(11, 21)
+    test_decode "inclusive", "'[10,20]'::int4range", Range.new(10, 21)
+    test_decode "inclusive", "'[10,20)'::int4range", Range.new(10, 20)
+  end
```

Comment:
There was recently a regression on a different part because we weren't testing negative numbers, so I think we should make sure to include at least one test with something like '[-14,-5]'

Reply:
Good call, I've added that test.